test/reload_evolution-luatest/reload_router_test.lua

```diff
@@ -0,0 +1,277 @@
+local t = require('luatest')
+local vtest = require('test.luatest_helpers.vtest')
+local os = require('os')
+
+-- Legacy utils
+local lutil = require('test.lua_libs.util')
+local lgit_util = require('test.lua_libs.git_util')
+
+local g = t.group('reload_router')
+local cfg_template = {
+    sharding = {
+        {
+            replicas = {
+                replica_1_a = {
+                    master = true,
+                },
+                replica_1_b = {},
+            },
+        },
+        {
+            replicas = {
+                replica_2_a = {
+                    master = true,
+                },
+            },
+        },
+    },
+    bucket_count = 100
+}
+
+local global_cfg
+
+g.before_all(function()
+    global_cfg = vtest.config_new(cfg_template)
+
+    -- We cannot work at the same directory, where the test is.
+    rawset(_G, 'vshard_copy_path', lutil.BUILDDIR ..
+           '/test/var/vshard_git_tree_copy_router_luatest')
+    -- Cleanup the dir after previous build
+    os.execute('rm -rf ' .. _G.vshard_copy_path)
```

Comment:
This shouldn't be needed, dangerous. Please, try to keep all files and folders local to the folder created for this run by luatest. I know it is done this way in `lua_libs/util.lua`, but `util.lua` is basically an anti-pattern how things should be done. I also know that I did it, now I see my design mistakes clearly.

Reply:
Now `VARDIR` is used